wallet-new/src/Cardano/Wallet/API/V1/Swagger/Example.hs

```diff
@@ -46,8 +46,25 @@ instance Example a => Example (WalletResponse a) where
                              <*> pure SuccessStatus
                              <*> example
 
+-- | We have a specific 'Example' instance for @'V1' 'Address'@ because we want
+-- to control the length of the examples. It is possible for the encoded length
+-- to become huge, up to 1000+ bytes, if the 'UnsafeMultiKeyDistr' constructor
+-- is used. We do not use this constructor, which keeps the address between
+-- ~80-150 bytes long.
+instance Example (V1 Address) where
+    example = fmap V1 . Core.makeAddress
+        <$> arbitrary
+        <*> arbitraryAttributes
+      where
+        arbitraryAttributes =
+            Core.AddrAttributes
+                <$> arbitrary
+                <*> oneof
+                    [ pure Core.BootstrapEraDistr
+                    , Core.SingleKeyDistr <$> arbitrary
+                    ]
```

Comment:
This is where the change is made that reduces the address length @erikd  :)

Reply:
Ok, so that is why it wasn't obvious. Its totally not obvious :).

Reply:
I'll annotate the PR better next time. I always like to leave a self-review but sometimes I'm too lazy.